dds/DCPS/RTPS/Sedp.cpp

```diff
@@ -1749,7 +1752,9 @@ void Sedp::process_discovered_writer_data(DCPS::MessageId message_id,
       }
 
     } else if (qosChanged(iter->second.writer_data_.ddsPublicationData,
-                          wdata.ddsPublicationData)) { // update existing
+                          wdata.ddsPublicationData))
```

Comment:
I can't tell why this is changing.  Was there some automated reformatting run on it?  If so we can use git to restore it.

Reply:
Is this just an issue with the line 1755?

Reply:
I think there were a few of them, but this was more visible since the comment and the `{` moved.  I'd suggest reading through the overall diff (in the web interface or `git diff master`) and see if there are changes you didn't intend to have.